tests/conftest.py

```diff
@@ -428,18 +428,20 @@ def ts_diff_endings(example_reg_tsds):
 
 
 @pytest.fixture
-def df_with_nans_in_tails(example_df):
```

Comment:
Don't touch these fixtures, they are global and are used in many tests that you shouldn't fix in this PR
1. Make them df again
2. Roll back tests fro pipeline/stl/trend/detrend